Solver/Dynamic/CMakeLists.txt

```
@@ -66,6 +66,11 @@ gridkit_add_library(solvers_dyn
   LINK_LIBRARIES
     PUBLIC SUNDIALS::nvecserial
     PUBLIC SUNDIALS::idas
+    PUBLIC SUNDIALS::sunlinsolklu
+    PUBLIC suitesparseconfig
+    PUBLIC klu
+    PUBLIC amd
+    PUBLIC colamd
   OUTPUT_NAME
```

Remove linkers for KLU, AMD, etc. These will just create dangling links `-lklu`, `-lamd`, etc., without setting library path. If Sundials is built with KLU support, its CMake.config will link the right libraries.
```suggestion
  LINK_LIBRARIES
    PUBLIC SUNDIALS::nvecserial
    PUBLIC SUNDIALS::idas
    PUBLIC SUNDIALS::sunlinsolklu
  OUTPUT_NAME
```